lib/add-properties.ts

```diff
@@ -0,0 +1,12 @@
+const traverse = require('traverse'),
+    _ = require('lodash');
+
+function addProperties(features: GeoJSON.Feature[], props: Object): GeoJSON.FeatureCollection {
```

Comment:
Please use a `=>` function here.

Reply:
We don't need to write a new function to do this. Use `PropertyManager.add` instead.